_includes/tutorials/console-consumer-produer-basic/kafka/markup/dev/consume-topic-from-beginning.adoc

```diff
@@ -0,0 +1,18 @@
+Next let's open up a console consumer again, but this time you'll ready everthing that your producer has sent to the topic. you created in the previous step.
+
+Run this command in a new terminal to start the consumer and note the additional property `--from-beginning`:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/console-consumer-from-beginning.sh %}</code></pre>
++++++
+
+After the consumer starts you should see the following output in a few seconds:
+
++++++
+<pre class="snippet"><code class="shell">{% include_raw tutorials/console-consumer-produer-basic/kafka/code/tutorial-steps/dev/expected-output-step-two.txt %}</code></pre>
++++++
+
+
+One word of caution with using the `--from-beginning` flag. As the name implies this setting forces the consumer retrieve _**every record**_ currently on the topic.  So it's best to use when testing  and learning and not on a production topic.
+
+Again, once you've recieved all records, close this console consumer by entering a `CTRL+C`.
```

Comment:
For consideration: there is also `--max-messages` flag.  I think `ctrl-c` may still be the better option, but just wanted to mention the flag.

Reply:
yeah the files for the test harness use the `--max-messages` I considered using that here, but opted for `cntrl-c` in case uses wanted to play around some more.